packages/special-pages/index.mjs

```diff
@@ -67,20 +72,20 @@ for (const [pageName, injectNames] of Object.entries(support)) {
             if (job === 'copy') {
                 copyJobs.push({
                     src: pageSrc,
-                    dest: pageOutputDirectory
+                    dest: pageOutputDirectory,
+                    injectName
                 })
             }
             if (job === 'build-js') {
-                const jsSrc = join(pageSrc, 'js', 'index.js')
-                if (!existsSync(jsSrc)) {
-                    errors.push(`${jsSrc} does not exist`)
-                    continue
-                }
-                const jsDest = join(pageOutputDirectory, 'js', 'index.js')
+                const entryPoints = [
+                    join(pageSrc, 'js', 'index.js'),
+                    join(pageSrc, 'js', 'inline.js')
+                ].filter(pathname => existsSync(pathname));
```

Comment:
this is allowing an additional entry point, called `inline.js` which can be used as a blocking request in HTML documents, useful for setting up environment stuff (since it has access to `import.meta` vars). The idea is that this would just be a few lines for now, but could be used to tree-shake more stuff in the future like messaging implementations etc.